airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/CdcPartitionsCreator.kt

```diff
@@ -113,8 +138,4 @@ class CdcPartitionsCreator<T : Comparable<T>>(
         log.info { "Current position '$lowerBound' does not exceed target position '$upperBound'." }
         return listOf(partitionReader)
     }
-
-    companion object {
-        var CDCNeedsRestart: Boolean = false
```

Comment:
This has been replaced by an `AtomicBoolean` in the cdc partitions creator factory class.